scipy/stats/tests/test_distributions.py

```diff
@@ -2831,6 +2831,70 @@ def test_minimum_data_zero_gh17801(self):
             _assert_less_or_close_loglike(dist, data, dist.nnlf)
 
 
+class TestPowerLogNorm:
+
+    # reference values were computed via mpmath
+    # from mpmath import mp
+    # mp.dps = 80
+    # def powerlognorm_sf_mp(x, c, s):
+    #     x = mp.mpf(x)
+    #     c = mp.mpf(c)
+    #     s = mp.mpf(s)
+    #     return float(mp.ncdf(-mp.log(x) / s)**c)
+
+    @pytest.mark.parametrize("x, c, s, ref",
+                             [(100, 20, 1, 1.9057100820561928e-114),
+                              (1e-3, 20, 1, 0.9999999999507617),
+                              (1e-3, 0.02, 1, 0.9999999999999508),
+                              (1e22, 0.02, 1, 6.50744044621611e-12)])
+    def test_sf(self, x, c, s, ref):
+        assert_allclose(stats.powerlognorm.sf(x, c, s), ref, rtol=1e-13)
+
+    # reference values were computed via mpmath using the survival
+    # function above. For inverse sf, we just have to reverse q / ref
```

Comment:
```suggestion
    # function above.
```
It doesn't look like that's what was done. If it were, just define the array of test cases above and share them.

Reply:
The inverse survival function is way less accurate than the survival function, so sharing test cases is not really possible.

Reply:
That's fine. The primary suggestion was to modify the comment.

Reply:
I will push the modification I was looking for. It will simply condense the test code to a block at the beginning.